src/components/field/index.tsx

```diff
@@ -27,6 +28,8 @@ export interface FieldDragSourceProps {
 }
 
 export interface FieldPropsBase {
+  filters?: Array<RangeFilter | OneOfFilter>;
```

Comment:
`filters` are only need if `filterShow` is true.  So you don't really need `filters` in many instances of Field. 

If you want to keep limit one filter per one field, One design option that would make the parameters of Field less exploded.  
```
filterShow?: boolean
```
to be

```
filterShow?: {
   filters:  Array<RangeFilter | OneOfFilter>
} 
```

This way it will be pretty clear that if you want to filter, pass in `filters` as it's needed for filter.